tests/encode/test1_suit/CMakeLists.txt

```diff
@@ -9,51 +9,25 @@ cmake_minimum_required(VERSION 3.13.1)
 set(PROJECT_NAME test1_suit)
 include(../../cmake/test_template.cmake)
 
-set(py_command_e
+set(py_command
   zcbor
   -c ${CMAKE_CURRENT_LIST_DIR}/../../cases/manifest3.cddl
   code
-  --output-cmake ${PROJECT_BINARY_DIR}/manifest3_encode.cmake
+  --output-cmake ${PROJECT_BINARY_DIR}/manifest3.cmake
   -t
   SUIT_Outer_Wrapper
   SUIT_Command_Sequence
   SUIT_Authentication_Wrapper
-  -e
-  --oc ${PROJECT_BINARY_DIR}/src/manifest3_encode.c
-  --oh ${PROJECT_BINARY_DIR}/include/manifest3_encode.h
-  --output-h-types ${PROJECT_BINARY_DIR}/include/manifest3_types.h
+  -e -d
   ${bit_arg}
   )
 
-set(py_command_d
-  zcbor
-  -c ${CMAKE_CURRENT_LIST_DIR}/../../cases/manifest3.cddl
-  code
-  --output-cmake ${PROJECT_BINARY_DIR}/manifest3_decode.cmake
-  -t
-  SUIT_Outer_Wrapper
-  SUIT_Command_Sequence
-  SUIT_Authentication_Wrapper
-  -d
```

Comment:
Does this PR remove all use of separate -e and -d calls like we do here? If not, it should IMO.